lib/pbench/test/unit/server/query_apis/test_controllers_list.py

```diff
@@ -120,7 +120,7 @@ def test_bad_dates(self, client, server_config, user_ok, pbench_token):
         )
 
     @pytest.mark.parametrize(
-        "user", ("drb", "", "no_user", None),
+        "user", ("drb", "no_user", None),
```

Comment:
I realize that you removed special handling of empty strings from the CUT...are you sure you want to remove the scenario from the test??  I mean, the CUT should still handle that case in predictable fashion....